pg_lakehouse/src/fdw/trigger.rs

```diff
@@ -251,6 +251,12 @@ fn construct_alter_table_statement(
         })
         .collect();
 
+    let table_name = if preserve_casing {
```

Comment:
Is this if/else necessary? Can we just always wrap `table_name` in quotes?

Reply:
I was thinking of that, but, if someone does not set a preserve_case I feel like that could cause a case sensitivity discrepancy. I'm definitely okay with adjusting it to that but it will make preserve_casing on the table name redundant since it will always preserve casing, unless, that's the expectation.

Reply:
I think there might not be any discrepancy. If the user creates a table without double quotes, then it will automatically be lowercased, in which case `table_name` will be lowercased and wrapping it in double quotes won't do anything. And if they do create the table with quotes, then it should always be wrapped in quotes here.

Reply:
Sounds good! I'll go ahead and adjust 👍

Reply:
@rebasedming updates made 👍